aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -342,6 +352,75 @@ module aptos_framework::account {
         rotation_capability
     }
 
+    public entry fun offer_signer_capability(account: &signer, signer_capability_sig_bytes: vector<u8>, account_scheme: u8, account_public_key_bytes: vector<u8>, recipient_address: address) acquires Account {
```

Comment:
1. let's call it `offer_signer_authority`
2. signer_capability_sig_bytes rename to og_signer_proof_bytes?

Reply:
think we are going to defer the `SignerAuthority` solution to the future / longer-term plans for now

Reply:
could you help us understand what the plan is now? @davidiw  
I thought we still want to introduce signer authority and NOT to deprecate existing usage of signer_capability.

Reply:
welcome to the hell of naming :)

@areshand is right in that you cannot retrieve this capability, so it remains an authority.
@chloeqjz is right in that this does create a capability, but we only allow it to be used as an authority.

Also we named everything else capability.

My gut feeling is that we should probably continue along with capability but note that it is currently only useful as an authority.